lib/Models/CompositeCatalogItem.ts

```diff
@@ -39,8 +39,12 @@ export default class CompositeCatalogItem extends MappableMixin(
     );
   }
 
-  protected forceLoadMetadata(): Promise<void> {
-    return Promise.resolve();
+  protected async forceLoadMetadata(): Promise<void> {
+    await Promise.all(
+      this.memberModels
+        .filter(CatalogMemberMixin.isMixedInto)
+        .map(model => model.loadMetadata())
+    );
```

Comment:
Also synchronously calls load functions inside forceLoad. I don't remember if that's a problem or not.

Reply:
I'm not really sure how to deal with this